core/test/unit/services/webhooks_spec.js

```diff
@@ -6,6 +6,7 @@ var _ = require('lodash'),
 
     // Stuff we test
     webhooks = rewire('../../../server/services/webhooks'),
+    webhooksListen = rewire('../../../server/services/webhooks/listen'),
```

Comment:
Not sure if this is the best way to rewire this, or if there is a better way to rewire packaged module🤔

Reply:
Updated, the object approach like stub worked much better